test/parallel_api/algorithm/alg.modifying.operations/transform_unary.pass.cpp

```diff
@@ -91,11 +117,13 @@ main()
     test<std::int32_t, std::int32_t>();
     test<std::int32_t, float32_t>();
     test<std::uint16_t, float32_t>();
-    test<float32_t, float64_t>();
```

Comment:
Is there a reason to remove the float32->float64 test?  Just time?

Reply:
I think such case is redundant, because we already have cases with different  in/out types.